wg-osdk/README.md

```diff
@@ -11,6 +11,8 @@ To understand how this file is generated, see https://git.k8s.io/community/gener
 To organize and steer the development of the [Operator SDK](https://github.com/operator-framework/operator-sdk) and related projects.
 
 ## Meetings
+* Operator SDK Grooming Meeting: [Wednesdays at 11:00 PT (Western Time)](https://docs.google.com/document/d/1ujWb-rSJ4JWeHLVxK0WS5ZuSJgeESG42MDeYjSl9Q6U/edit#) (triweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=PT%20%28Western%20Time%29).
+* Operator SDK Issue Triage: [Mondays at 11:00 PT (Western Time)]() (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=PT%20%28Western%20Time%29).
```

Comment:
Is there a way to generate this without a URL?

Reply:
Not that I know of, perhaps @njhale can comment.